### PR TITLE
Adding middle pricing tier for FR

### PR DESCRIPTION
**Pull request by jamesshedden**
Pending final content from @adeclosset, but will look something a bit like this:

image

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

File: app/css/components/pricing-options.scss [hunk 1]
```diff
@@ -34,3 +34,11 @@
 .pricing-options--small {
   width: 470px;
 }
+
+.pricing-options__longer-heading {
+  line-height: 34px;
```
Comment (jamesshedden) [inline app/css/components/pricing-options.scss]: Don't need this anymore :scissors:

**Comment by adeclosset** [comment]
@jamesshedden - All good here too. Been a better boy here and I've updated the branch before committing new stuff. Should be all good?

**Comment by jamesshedden** [comment]
@adeclosset Taking a look :+1:

**Pull request merged**